<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceReconViewWidget</class>
 <widget class="QWidget" name="SurfaceReconViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>1202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>1) Choose input nodes</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>  Left channel:</string>
        </property>
        <property name="buddy">
         <cstring>LeftChannelNodeNameComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="LeftChannelNodeNameComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">choose or edit node name</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>  Right channel:</string>
        </property>
        <property name="buddy">
         <cstring>RightChannelNodeNameComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="RightChannelNodeNameComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <item>
         <property name="text">
          <string notr="true">choose or edit node name</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pick a data storage node from the list or enter its name. The node should have an image attached to it.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>2) Output node</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>  Output node:</string>
        </property>
        <property name="buddy">
         <cstring>OutputNodeNameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="OutputNodeNameLineEdit">
        <property name="text">
         <string>stereo matching output</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Type a name for the result node. If it does not exist it is created. If it exists a point cloud is attached to it. If you leave it empty a suitable name will be generated.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="outputoptionsgroupbox">
        <property name="title">
         <string>Show options</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QWidget" name="outputoptionsdummypanel" native="true">
           <layout class="QFormLayout" name="formLayout_5">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="GeneratePointCloudRadioBox">
              <property name="text">
               <string>Generate point cloud</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="GenerateDisparityImageRadioBox">
              <property name="text">
               <string>Generate disparity image</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>3) Choose method</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>  Method:</string>
        </property>
        <property name="buddy">
         <cstring>MethodComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="MethodComboBox">
        <item>
         <property name="text">
          <string>Sequential Quasi Dense Matching</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parallel Quasi Dense Matching</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Real-time Quasi Dense Matching</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Each method has different behaviour regarding noise susceptibility, run-time performance, suitability for high-resolution images, etc.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="DoItButton">
        <property name="text">
         <string>DO IT NOW</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>outputoptionsgroupbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>outputoptionsdummypanel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
